email-utils.js

```diff
@@ -50,7 +50,8 @@ const EmailUtils = {
     }
 
     const emailContext = Object.assign({
-      SERVER_URL: AppConstants.SERVER_URL
+      SERVER_URL: AppConstants.SERVER_URL,
+      layout: aTemplate
```

Comment:
😆 : heh, I forgot about the `aVar` naming convention we inherited from Firefox JS code. I don't even remember what the `a` stands for anymore.

Reply:
Wonder no more!

a=argument

---

Per https://firefox-source-docs.mozilla.org/code-quality/coding-style/coding_style_js.html#prefixes:

> #### Prefixes
> - k=constant (e.g. `kNC_child`). Not all code uses this style; some uses `ALL_CAPS` for constants.
> - g=global (e.g. `gPrefService`)
> - a=argument (e.g. `aCount`)
> - JavaScript Specific Prefixes
>     - _=member (variable or function) (e.g. `_length` or `_setType(aType)`)
>    - k=enumeration value (e.g. `const kDisplayModeNormal = 0`)
>    - on=event handler (e.g. `function onLoad()`)
>    - Convenience constants for interface names should be prefixed with `nsI`: